spec/code_hunk_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'spec_helper'
+
+RSpec.describe Airbrake::CodeHunk do
+  describe "#to_h" do
+    context "when a file is empty" do
+      subject { described_class.new(fixture_path('empty_file.rb'), 1).to_h }
+
+      it { is_expected.to eql({}) }
+    end
+
+    context "when a file doesn't exist" do
+      subject { described_class.new(fixture_path('banana.rb'), 1).to_h }
+
+      it { is_expected.to be_nil }
+    end
```

Comment:
Why not to throw an exception (return error)?

Reply:
What would it buy us?

Reply:
AFAIK that is how errors are returned in Ruby.

Reply:
If you have examples, I'd look at them. I don't think that's how it works. Either way, it's not a big deal because in real world backtraces would unlikely point to non existing files.